How to enable serde feature on wgpu-types through wgpu-rs ?

wgpu-types has a feature to allow (de)serialization for types, descriptors and enums. 
How can I enable it through wgpu-rs ?

Well I've tried it but the no magic happened :( 

in Cargo.toml
```
wgpu-types = {git = "https://github.com/gfx-rs/wgpu", features = ["serde"]}
wgpu = {git = "https://github.com/gfx-rs/wgpu-rs.git", branch = "master"}
```

compile result:
```
 Compiling gfx-backend-vulkan v0.5.6
   Compiling wgpu-types v0.5.0 (https://github.com/gfx-rs/wgpu#d4532350)
   Compiling wgpu-types v0.5.0 (https://github.com/gfx-rs/wgpu?rev=5c172dd4756aa152b4f3350e624d7b1b5d24ddda#5c172dd4)
   Compiling wgpu-core v0.5.0 (https://github.com/gfx-rs/wgpu?rev=5c172dd4756aa152b4f3350e624d7b1b5d24ddda#5c172dd4)
   Compiling wgpu v0.5.0 (https://github.com/gfx-rs/wgpu-rs.git#49640d24)
   Compiling shine-game v0.1.0 (D:\work\shine-backend\game\game)
error[E0277]: the trait bound `wgpu_types::VertexFormat: config::_IMPL_DESERIALIZE_FOR_Config::_serde::Serialize` is not satisfied             
 --> game\src\render\pipeline_descriptor.rs:8:14
  |
8 |     Position(wgpu::VertexFormat),
  |              ^^^^^^^^^^^^^^^^^^ the trait `config::_IMPL_DESERIALIZE_FOR_Config::_serde::Serialize` is not implemented for `wgpu_types::VertexFormat`
```

It seems as it tries to compile two instance with different version of the package.
One for my use: d4532350 is the current master head for wgpu
Another used by the wgpu-rs

If I add the `rev = "5c172dd4756aa152b4f3350e624d7b1b5d24ddda"` the feature is picked up cargo correctly but it is not too ergonomic. So It could be a workaround if the new feature for wgpu-rs is rejected.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,6 +18,7 @@ all-features = true
 [features]
 default = []
 trace = ["wgc/trace"]
+replay = ["wgc/replay"]
 subscriber = ["wgc/subscriber"]
 # Make Vulkan backend available on platforms where it is by default not, e.g. macOS
 vulkan = ["wgc/gfx-backend-vulkan"]

